src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,179 @@ Keywords:
 
 # Release notes
 
+## March 20, 2024
+
+### iOS Mobile Core 5.x and compatible extensions
+
+New major version of the Mobile Core SDK for iOS has been released along with updates to other extensions. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 12.0 and tvOS 12.0 (for extensions which support tvOS).
+* Include XCFrameworks built with Xcode 15.0.1 with the GitHub release.
+* Include a privacy manifest for AEPCore and AEPServices.
+
+Note that all these extensions must be updated together.
+
+* iOS Mobile Core 5.0.0
+* iOS Signal 5.0.0
+* iOS Lifecycle 5.0.0
+* iOS Identity 5.0.0
+* iOS Profile 5.0.0
+* iOS Adobe Experience Platform Assurance 5.0.0
+* iOS Adobe Experience Platform Edge Network 5.0.0
+* iOS Identity for Edge Network 5.0.0
+* iOS Consent for Edge Network 5.0.0
+* iOS Media for Edge Network 5.0.0
+* iOS Edge Bridge 5.0.0
+* iOS Adobe Journey Optimizer 5.0.0
+* iOS Adobe Journey Optimizer - Decisioning 5.0.0
+* iOS Places Service 5.0.0
+* iOS Adobe Analytics 5.0.0
+* iOS Adobe Analytics - Media Analytics for Audio & Video 5.0.0
+* iOS Adobe Target 5.0.0
+* iOS Adobe Campaign Standard 5.0.0
+* iOS Adobe Campaign Classic 5.0.0
+* iOS Adobe Audience Manager 5.0.0
+
+### iOS Messaging 5.0.0
+
+Major version update for [Adobe Journey Optimizer](../../edge/adobe-journey-optimizer/index.md) for Adobe Experience Platform Mobile SDKs on iOS compatible with Mobile Core 5.0.0. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 12.0.
+* Include XCFrameworks built with Xcode 15.0.1 with the GitHub release.
+* Added `handleNotificationResponse(_:urlHandler:closure:)` API to provide more control over URL handling from `UNNotificationResponse` objects.
+  * Removed deprecated public API `Messaging.handleNotificationResponse(_:applicationOpened:withCustomActionId:)`.
+  * Removed public API `Messaging.handleNotificationResponse(_:closure:)` API.
```

Comment:
for consistency should this be Removed deprecated public API ...

Reply:
I am not sure if this was a deprecated API. @sbenedicadb